sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -445,25 +458,28 @@ public EventHubClient buildClient() {
         return new EventHubClient(client, connectionOptions);
     }
 
-    private static EventHubAsyncClient buildAsyncClient(ConnectionOptions connectionOptions) {
+    private EventHubAsyncClient buildAsyncClient(ConnectionOptions connectionOptions) {
```

Comment:
`buildAsyncClient()` overload should be package-private and `buildClient()` should be deleted.

Reply:
Oh yeah, I thought I had reverted this. I had used it to verify samples, so users couldn't copy code that was broken.

Reply:
Oh, this is the private one used by the builder, there is a package-private no parameter one